ansible/README.md

```diff
@@ -0,0 +1,76 @@
+## MicroShift cluster automation with Ansible
+
+This directory contains a minimal workflow to build an Ansible runner container, prepare an inventory, install upstream MicroShift on target nodes, and join them into a single cluster.
+
+### Prerequisites
+- **Podman** installed on your workstation.
+- **SSH access** from your workstation to all target VMs.
+  - Public keys should be present in `~/.ssh/` and trusted by the target hosts.
+  - Passwordless sudo on the target hosts is recommended for automation.
+- Target hosts are reachable by DNS or IP and allow inbound SSH.
+- **Note:** This workflow assumes that your VMs are running **CentOS Stream 9** as the operating system.
+
+### Build the Ansible runner image (Podman)
+Build the container that bundles Ansible and playbooks in this repo:
+
+```bash
+podman build -f ansible-runner.Containerfile -t ansible-runner:latest .
+```
+
+### Create the Ansible inventory
+Create an `inventory` file in `cluster/` directory with your managed hosts. Example:
+
+```ini
+[remote]
+hostname-fqdn-pri ansible_connection=ssh ansible_user=ec2-user role=primary
+hostname-fqdn-sec ansible_connection=ssh ansible_user=ec2-user role=secondary
+```
+
+Notes:
+- `ansible_user` must exist on the host and have sudo privileges.
+- `role` is used by the playbooks to distinguish the primary node from secondaries.
+- This setup assumes SSH keys are available at `~/.ssh/` on your workstation.
+
+### Install latest upstream MicroShift on all nodes
+This installs MicroShift on each node independently.
```

Comment:
The `independently` word is confusing. Let's state that we're installing the first primary node of the cluster here.

Reply:
actually at this stage the node role doesnt make any difference , so ansible  installing  Microshift in all the nodes in parallel.